sdv/single_table/base.py

```diff
@@ -150,6 +156,9 @@ def __init__(
             self.metadata.tables[self._table_name]._updated = metadata._updated
 
         self.metadata.validate()
+
+        # Points to a metadata object that conserves the initialized status of the synthesizer
+        self._original_metadata = deepcopy(self.metadata)
```

Comment:
This needs to be set in line 162. `_check_metadata_updated` sets `self.metadata._updated` to False, which is how the `_original_metadata` attribute should be initialized.

Reply:
Yeah this makes sense!